orttraining/orttraining/core/graph/pipeline_transformer.cc

```diff
@@ -22,17 +23,20 @@ bool IsBackward(Node& node) {
   return (node.Description() == "Backward pass");
 }
 
+NodeArg& CreateInt64NodeArg(Graph& graph, std::string name) {
```

Comment:
>std::string [](start = 42, length = 11)

consider changing to `const std::string&` to avoid accidental change and string copy.

Reply:
Ok.